openedx/features/ondemand_email_preferences/models.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.auth.models import User
+from django.db import models
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+
+class OnDemandEmailPreferences(models.Model):
+    user = models.ForeignKey(User, db_index=True)
+    course_id = CourseKeyField(db_index=True, max_length=255, null=False)
```

Comment:
Let's add relation to course_overview table

Reply:
This is not needed as per discussion.